mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/core/data/root/impl/BranchRootImpl.java

```diff
@@ -0,0 +1,118 @@
+package com.gentics.mesh.core.data.root.impl;
```

Comment:
This file has been actually moved from Core. Github treated in as created anew somehow.